apps/plasmo/src/components/Calendar/CalendarMonthYear.tsx

```
@@ -0,0 +1,179 @@
+    } else {
+      // Otherwise just increase the month
+      onMonthChange(String(currentMonthIndex + 1).padStart(2, "0"))
+    }
+  }
+
+  return (
+    <div className="plasmo-flex plasmo-items-center plasmo-gap-3">
+      <button
+        onClick={handlePreviousMonth}
+        className="plasmo-p-1 plasmo-rounded hover:plasmo-bg-gray-700 focus:plasmo-outline-none focus:plasmo-ring-2 plasmo-ring-blue-500"
+        aria-label="Previous month">
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)